ts/boot/configureInstabug.ts

```diff
@@ -29,6 +29,13 @@ export const initialiseInstabug = () => {
   );
 };
 
+export const setInstabugUserAttribute = (
+  attributeKey: string,
```

Comment:
to make this really useful you could use it to type check the `attributeKey` with

```suggestion
  attributeKey: "backendVersion" | "activeScreen",
```